src/Http/Http.Extensions/test/RequestDelegateGenerator/RequestDelegateGeneratorTestBase.cs

```diff
@@ -25,6 +25,10 @@ namespace Microsoft.AspNetCore.Http.Generators.Tests;
 
 public abstract class RequestDelegateGeneratorTestBase : LoggedTest
 {
+    // Change this to true and run tests in development to regenerate baseline files.
+    // Then: cp artifacts/bin/Microsoft.AspNetCore.Http.Extensions.Tests/Debug/net8.0/RequestDelegateGenerator/Baselines/* src/Http/Http.Extensions/test/RequestDelegateGenerator/Baselines
+    public bool RegenerateBaselines => false;
```

Comment:
Good addition. In the `CompareLines(...)` method we should probably add logic ignore lines with the code analysis attribute so that when folks follow the instruction above they then don't need to do a regex/replace on the attributes with the `%%...%%`.

Reply:
... I see you've taken care of this below :)